### PR TITLE
Updating the Bash documentation page

### PR DESCRIPTION
**Pull request by cstackpole**
Many of the pages on the Bash page (http://exercism.io/languages/bash) are missing. They have helpful hints on what needs to be corrected/added. These commits address those helpful hints as a start point.

### CODE REVIEW

File: docs/INSTALLATION.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Installation
```
Comment (kytrinyx) [inline docs/INSTALLATION.md]: The docs shouldn't have titles, since the website adds section titles (and this will result in double titles).

**Comment by kytrinyx** [comment]
Thanks!

I have just one comment about the format of the docs themselves (see inline).

The build is breaking on the `configlet .` command.
I think the fix is going to be to put `docs` into the `"ignored"` section of the `config.json` file. 
If you want to run `configlet` locally, you can first run `bin/fetch-configlet` and then `configlet .` (notice the dot) from the xbash directory.

**Comment by cstackpole** [comment]
Thanks for the feedback. Just verified that configlet works. Cleaning up the headers now.

**Comment by kytrinyx** [comment]
Thanks!

**Pull request merged**